chapters/hpmor-chapter-001.tex

```diff
@@ -1,176 +1,175 @@
 
-“Darling,” Michael said gently, “you got sick, you gained some weight while resting in bed, and your skin cleared up on its own. Or being sick made you change your diet—”
+“Darling,” Mikael said gently, “you got sick, you gained some weight while resting in bed, and your skin cleared up on its own. Or being sick made you change your diet—”
 
-“She was a witch,” Petunia repeated. “I saw it.”
+“She was a witch,” Petunja repeated. “I saw it.”
 
-“Petunia,” Michael said. The annoyance was creeping into his voice. “You \emph{know} that can’t be true. Do I really have to explain why?”
+“Petunja,” Mikael said. The annoyance was creeping into his voice. “You \emph{know} that can’t be true. Do I really have to explain why?”
 
-Petunia wrung her hands. She seemed to be on the verge of tears. “My love, I know I can’t win arguments with you, but please, you have to trust me on this—”
+Petunja wrung her hands. She seemed to be on the verge of tears. “My love, I know I can’t win arguments with you, but please, you have to trust me on this—”
 
 “\emph{Dad! Mum!}”
 
-The two of them stopped and looked at Harry as though they’d forgotten there was a third person in the room.
+The two of them stopped and looked at Hari as though they’d forgotten there was a third person in the room.
 
-Harry took a deep breath. “Mum, \emph{your} parents didn’t have magic, did they?”
+Hari took a deep breath. “Mum, \emph{your} parents didn’t have magic, did they?”
 
-“No,” Petunia said, looking puzzled.
+“No,” Petunja said, looking puzzled.
 
-“Then no-one in your family knew about magic when Lily got her letter. How did \emph{they} get convinced?”
+“Then no-one in your family knew about magic when Lili got her letter. How did \emph{they} get convinced?”
 
-“Ah…” Petunia said. “They didn’t just send a letter. They sent a professor from Hogwarts. He—” Petunia’s eyes flicked to Michael. “He showed us some magic.”
+“Ah…” Petunja said. “They didn’t just send a letter. They sent a professor from Hogwarts. He—” Petunja’s eyes flicked to Mikael. “He showed us some magic.”
 
-“Then you don’t have to fight over this,” Harry said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogwarts professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”
+“Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogwarts professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”
```

Comment:
```suggestion
“Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogŭartso professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”
```